packages/devtools_app/test/flutter/wrappers.dart

```diff
@@ -45,18 +46,26 @@ Widget wrapWithControllers(
   );
 }
 
-/// Sets the size of the app window under test to [windowSize].
-///
-/// This must be reset on after each test invocation that calls
-/// by using [resetWindowSize].
-Future<void> setWindowSize(Size windowSize) async {
+/// Runs a test with the size of the app window under test to [windowSize].
+@isTest
+void testWidgetsWithSize(
```

Comment:
nit: rename to `testWidgetsWithWindowSize` as `withSize` makes it seem like size is referring to the widget, not the window

Reply:
Done.

Reply:
done.